Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverOrbital.cs

```diff
@@ -0,0 +1,182 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// SolverObjectLock provides a solver that follows the TrackedObject/TargetTransform. Adjusting "LerpTime"
+    /// properties changes how quickly the object moves to the TrackedObject/TargetTransform's position.
+    /// </summary>
+    public class SolverOrbital : Solver
+    {
+        #region public enums
+        public enum OrientationReferenceEnum
+        {
+            /// <summary>
+            /// Use the tracked object's pitch, yaw, and roll
+            /// </summary>
+            FollowTrackedObject,
+            /// <summary>
+            /// Face toward the tracked object
+            /// </summary>
+            FaceTrackedObject,
+            /// <summary>
+            /// Orient towards SolverHandler's tracked object or TargetTransform
+            /// </summary>
+            YawOnly,
+            /// <summary>
+            /// Leave the object's rotation alone
+            /// </summary>
+            Unmodified,
+            /// <summary>
+            /// Orient toward Camera.main instead of SolverHandler's properties.
+            /// </summary>
+            CameraFacing,
+            /// <summary>
+            /// Align parallel to the direction the camera is facing 
+            /// </summary>
+            CameraAligned,
+        }
+        #endregion
+
+        #region public members
+        [Tooltip("The desired orientation of this object. Default sets the object to face the TrackedObject/TargetTransform. CameraFacing sets the object to always face the user.")]
+        [SerializeField]
+        private OrientationReferenceEnum orientation = OrientationReferenceEnum.FollowTrackedObject;
+
+        public OrientationReferenceEnum Orientation
+        {
+            get { return orientation; }
+            set { orientation = value; }
+        }
+
+        [Tooltip("XYZ offset for this object in relation to the TrackedObject/TargetTransform. Mixing local and world offsets is not recommended.")]
+        [SerializeField]
+        private Vector3 localOffset = new Vector3(0,-1,1);
+
+        public Vector3 LocalOffset
+        {
+            get { return localOffset; }
+            set { localOffset = value; }
+        }
+
+        [Tooltip("XYZ offset for this object in worldspace, best used with the YawOnly orientation. Mixing local and world offsets is not recommended.")]
+        [SerializeField]
+        private Vector3 worldOffset = Vector3.zero;
+
```

Comment:
Let's add summary tags with the text from the tooltips to match for all properties